openedx/adg/lms/webinars/admin.py

```diff
@@ -109,8 +109,10 @@ def get_fields(self, request, obj=None):
         Override `get_fields` to dynamically set fields to be rendered.
         """
         fields = super().get_fields(request, obj)
-        if not obj:
+
+        if not obj or not obj.is_published:
```

Comment:
The following looks more readable to me. If you agree, we can make the change otherwise this looks good too.

```
is_webinar_published = getattr(obj, 'is_published', False)
if is_webinar_published:
```